src/Row.ts

```diff
@@ -37,15 +37,44 @@ class Row extends RowBase<RowProperties> {
 				classes: this.classes(tableCss.table, css.rowTable)
 			}, [
 				v('tr', columns.map((column) => {
-					const { id, field } = column;
+					const { field, id } = column;
+
+					let value: any;
+					if (column.get) {
+						// Get the value from the column callback
+						value = column.get(item, column);
+					}
+					else if (!column.render) {
+						// Get the value using a property lookup on the item data
+						value = item.data[ field || id ];
+					}
+					else {
+						// This column has no value
+						value = '';
+					}
+
+					let content: DNode;
+					if (column.render) {
```

Comment:
I've been going back and forth on this, and it's been discussed in the referenced issue. My intent with render was to have a function that indicates it will not be consuming any data - adding an action button, for example. Absent this, we will still attempt to read a value from the underlying data - which isn't horrible but creates a side-effect that may be unexpected and could (though it would be unlikely) create problems.

Some alternatives I'd been considering:

* Add something like an enum that can be assigned to `field` that would indicate no value should be read from the data object
* Allow `get` to be either a function or a value - perhaps rename this property to `value`

 What do you think of these alternatives?